### PR TITLE
Changed README.md

### PR DESCRIPTION
**Pull request by YawarValeriano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Pokedex-ios
 iOS implementation for PokeAPI pokedex
+
+Testing pull request in README
```
Comment (jmacboy) [inline README.md]: Also add the link to the wiki page

**Review by jmacboy — CHANGES_REQUESTED** [review]
Please add more information

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # Pokedex-ios
 iOS implementation for PokeAPI pokedex
```
Comment (Rodrigo-Schar) [inline README.md:2]: add link to the figma file

**Review by Rodrigo-Schar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (YawarValeriano) [reply README.md]: It's not necessary, we already have the link in repo

File: README.md (hunk 2 above)
Comment (YawarValeriano) [reply README.md:2]: Done

**Review by jmacboy — APPROVED** [review]

**Review by Rodrigo-Schar — APPROVED** [review]

**Pull request merged**